admin_guide/backup_restore.adoc

```diff
@@ -0,0 +1,115 @@
+= Backup and Restore
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: how to backup and restore at project and cluster level
+:keywords: backup, restore
+
+toc::[]
+
+
+// REVIEWERS: READ THIS!
+//
+// In the following text, there are questions of the form:
+//   //??? QUESTION
+// Please feel free to make a line-comment to answer them, in addition
+// to any other (line-)comments on the correctness of the text.
+//
+// - Usually, the question pertains to the text preceding it.
+//   Questions pertaining to following text are explicitly noted.
+//
+// - There are bunch of questions at the end.
+//
+// Thanks for your cooperation on this (experimental) method of
+// refining the documentation.  Hopefully it will bear good fruit.
+
+== Overview
+
+In {product-title}, you can
+do _backup_, saving state to separate storage,
+and _restore_, recreating state from separate storage,
+at the cluster level.
+The full state of a cluster installation includes:
+- etcd data on each master
+- API objects
+//??? Anything else?
+//??? What about (non-master) nodes?
+
+[NOTE]
+Project-level backup/restore will be available in a future release.
+//??? Is this true?
+
+This section does not cover backup and restore of
+link:../install_config/persistent_storage/index.html[persistent storage].
+//??? But should it?
```

Comment:
From https://docs.openshift.com/enterprise/3.2/install_config/persistent_storage/index.html, or all of the sub sections you have the note: 

> High-availability of storage in the infrastructure is left to the underlying storage provider.

Because of this, no backup or restore should be needed for "storage" or "volumes" as high availability or backups / disaster recovery are left to the **underlying storage provider**. If we have docs, on backups / disaster recovery for supported storage providers, we might want to link to these (think Ceph, Gluster, Cinder, etc).